### PR TITLE
Clear browser storage between tests

### PR DESCRIPTION
**Pull request by gcandal**

### CODE REVIEW

File: src/ripe_rainbow/interactive/drivers.py [hunk 1]
```diff
@@ -203,6 +212,15 @@ def stop(self):
         self._flush_log()
         InteractiveDriver.stop(self)
 
+    def clear_cookies(self):
+        self.instance.delete_all_cookies()
+
+    def clear_local_storage(self):
+        self.instance.execute_script("localStorage.clear();")
```
Comment (joamag) [inline src/ripe_rainbow/interactive/drivers.py:219]: @gcandal this only clear the local storage for the current domain https://developer.mozilla.org/en-US/docs/Web/API/Storage/clear

**Comment by joamag** [comment]
It seams there's a way to get local storage https://sqa.stackexchange.com/questions/10466/how-to-clear-localstorage-using-selenium-and-webdriver but I couldn't find anything in python.

**Comment by joamag** [comment]
We could in alternative use the "hardcoded" method of going to chrome settings and clicking the clear cache 

```java
driver.get("chrome://settings/clearBrowserData");
Thread.sleep(10000);
driver.findElement(By.xpath("//*[@id='clearBrowsingDataConfirm']")).click();
```

**Comment by joamag** [comment]
Another idea is to clear localstorage whenever we jump between URLs !!!

**Comment by gcandal** [comment]
> It seems there's a way to get local storage https://sqa.stackexchange.com/questions/10466/how-to-clear-localstorage-using-selenium-and-webdriver but I couldn't find anything in python.
> 
> ```js
> driver.getSessionStorage().clear();
> driver.getLocalStorage().clear();
> ```
> 
> There's also another example
> 
> ```js
> driver.WebStorage.LocalStorage.Clear();
> ```

I was writing a comment saying why it isn't possible to do it in Python but I think I might've found a way, let me just test the idea.

**Comment by gcandal** [comment]
> > It seems there's a way to get local storage https://sqa.stackexchange.com/questions/10466/how-to-clear-localstorage-using-selenium-and-webdriver but I couldn't find anything in python.
> > ```js
> > driver.getSessionStorage().clear();
> > driver.getLocalStorage().clear();
> > ```
> > 
> > 
> > There's also another example
> > ```js
> > driver.WebStorage.LocalStorage.Clear();
> > ```
> 
> I was writing a comment saying why it isn't possible to do it in Python but I think I might've found a way, let me just test the idea.

Nevermind, it doesn't work. Even if they [have the command declared](https://github.com/SeleniumHQ/selenium/blob/7824d1d5991d340867ef1d8ef01be41df5c23ec9/py/selenium/webdriver/remote/command.py#L156), it doesn't work because it's not part of the [WebDriver spec](https://www.w3.org/TR/webdriver/#delete-all-cookies), it's there as legacy code from before the WebDriver days.

**Pull request merged**